### DISCUSSION TITLE
Create GitHub Security Advisory for Sanitize markdown preview

### DISCUSSION THREAD

**Discussion by SuperSandro2000** [pretalx/pretalx#1815] [Ideas]
Hi,
it would be nice if there would be a github security advisory for https://github.com/pretalx/pretalx/commit/58e9990d4e2c81d1eb9a23f77801efcf5155ac08 .

I don't want you to take the long journey for a CVE but just having that available and findable would have already helped me a lot 😅 

Also I think the [changelog](https://docs.pretalx.org/changelog/#v2024-2-0) could emphazise this a bit more

**Comment by rixx** [comment]
As far as I am aware, we'd have to start accepting vulnerability reports via GitHub in order to be able to publish advisories on here (or at least, the documented way to create and publish an advisory doesn't seem to exist for me unless I turn reporting on GitHub on). I'm very hesitant to tie pretalx more to GitHub infrastructure than it is already. Publishing information like this in more venues may be feasible for bigger projects with whole teams, but pretalx is something that I develop and run in my spare time, so I have to pick my battles.

The release blog post as well as the social media post emphasized that the latest release came with a fix for a security vulnerability and that an update was very much recommended. The changelog also states very openly:

>  The markdown preview posed a security vulnerability by allowing speakers and organisers to include unsafe JavaScript.

There's currently no way to mark vulnerabilities separately in the changelog. If anybody wants to go ahead and adjust the changelog code to improve this, I'm happy to accept a contribution :)

**Reply by SuperSandro2000** [reply]
I only just saw the blog post now 😅  https://pretalx.com/p/news/releasing-pretalx-2024-2-0/

> There's currently no way to mark vulnerabilities separately in the changelog. If anybody wants to go ahead and adjust the changelog code to improve this, I'm happy to accept a contribution :)

Just do a one time manual edit 😉
